packages/react/xr/src/xr.tsx

```diff
@@ -138,3 +141,20 @@ export function useXR<T = XRState>(
 ) {
   return useStore(useXRStore(), selector, equalityFn)
 }
+
+/**
+ * function for getting the object of a specific component from the xr controller model
+ */
+export function getXRControllerComponentObject(
```

Comment:
I was rather saying that this function should be moved into packages/xr/ so vanilla users can also leverage the function ;-)